src/prisma/binaries/platform.py

```diff
@@ -1,75 +1,125 @@
+import platform
 import re
-import sys
 import subprocess
-import platform as _platform
+import sys
 from functools import lru_cache
-from typing import Tuple
-
-
-def name() -> str:
-    return _platform.system().lower()
+from typing import Optional
 
+import distro  # type: ignore # pyright: reportMissingTypeStubs=false
```

Comment:
```suggestion
import distro  # type: ignore[import]
```
This is a subtle change but means that both mypy and pyright will not report an error for missing type stubs but will still type check it. Previously pyright would disregard distros typings. You could either make this change wherever distro is imported or add an entry to `_compat.py`:

```py
import distro as distro  # type: ignore[import]
```
I haven't tested this but it should work.